### PR TITLE
head (tail) function

### PR DESCRIPTION
**Pull request by Elaray1**

### CODE REVIEW

File: src/tailFunc.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const tail = arr => {
+  return arr.length === 0 ? 'Empty array!' : arr[arr.length-1]
```
Comment (AllDayAlone) [inline src/tailFunc.js:2]: image

you've written `last`

Let's rename it to last and return `undefined` if there are no elements

Also please handle a situation when user provide `null` as an argument (should also return undefined)

**Review by alldayalone — CHANGES_REQUESTED** [review]
@Elaray1 Hey, make changes please

File: test/tailFunc.spec.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  const arr3 = [100, 200, 300, 400, 500];
+  const arr4 = [];
+
+  // success
+  assert.equal(tail(arr1), 5);
+  assert.equal(tail(arr2), 50);
+  assert.equal(tail(arr3), 500);
+
+  // bad arguments
+  assert.equal(tail(arr4), 'Empty array!');
```
Comment (AllDayAlone) [inline test/tailFunc.spec.js:16]: should be undefined

File: src/tailFunc.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const tail = arr => {
```
Comment (AllDayAlone) [inline src/tailFunc.js:1]: please name this file `tail.js`

File: test/tailFunc.spec.js (hunk 2 above)
Comment (Elaray1) [reply test/tailFunc.spec.js]: changed

File: src/tailFunc.js (hunk 3 above)
Comment (Elaray1) [reply src/tailFunc.js]: renamed

File: src/traverseFunc.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+const forIn = require('lodash/forIn');
```
Comment (AllDayAlone) [inline src/traverseFunc.js:1]: this file should not be here

Comment (Elaray1) [reply src/traverseFunc.js]: deleted

File: test/tail.spec.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+const tail = require('../src/tailFunc');
+
+function test() {
+  const arr1 = [1, 2, 3, 4, 5];
+  const arr2 = [10, 20, 30, 40, 50];
+  const arr3 = [100, 200, 300, 400, 500];
+  const arr4 = [];
+
+  // success
+  assert.equal(tail(arr1), 5);
```
Comment (SobakaSlava) [inline test/tail.spec.js:11]: do we expect 5 as result? did you run the tests?

**Review by mikitasolo — CHANGES_REQUESTED** [review]

File: test/tail.spec.js (hunk 5 above)
Comment (Elaray1) [reply test/tail.spec.js:11]: changed

**Pull request merged**